### DISCUSSION TITLE
Moonray for Windows

### DISCUSSION THREAD

**Discussion by xXTR0YXx** [dreamworksanimation/openmoonray#95] [Q&A]
So, enough doodeling around with the LINUX build. 

Time to get serious and get a working windows build. Currently trying to assemble all the required dependencies and figuring out what lines of cmake need to be altered to make it work with Windows. Cloning the git worked (Only got an error for the dcc plugins folder but skipping that for now). Now trying to get all the dependencies downloaded and being build but I'm stuck with Boost which gives me an error when trying to build it.

Is there a way to skip that part, since I already have already downloaded a pre-compiled version of Boost 1.73.0 of windows?

**Comment by rgwilson68** [comment]
You can skip the build of anything in the dependencies CMakeLists.txt (in /building) by commenting it out, but be sure to adjust the dependency of the next uncommented target. If you install the pre-compiled version of boost to the standard location (don't know what that is on Windows...) CMake should automatically find it for the remaining dependencies and the main build.

**Comment by xXTR0YXx** [comment]
Having a hard time getting all the right dependencies compiling correctly for windows. Like Boost 1.73.0 doesn't work with Visual Studio 2022 because it's too old, so I went for a pre-built one, which workd. At the moment I am stuck with TBB or better OpenVDB trying to access a file from the TBB build that is there, but "can't be opened". I have also tried it with a pre-built TBB but same error for the same file.

Anyone else having success with this?

**Comment by Guscal** [comment]
any new progress?
